app/lib/run-playbook.sh

```diff
@@ -11,7 +11,11 @@ main() {
 	export BASEDIR=${BASEDIR:-.}
 
 	# Ensure we have HOME defined, otherwise set it manually
-	test -z "$HOME" && export HOME; HOME="$(getent passwd "$(id -un)" | cut -d: -f6)"
+	if [[ "$(uname)" == 'Darwin' ]]; then
```

Comment:
`[[` is incompatible with `sh`

Keep `test` instead.

```suggestion
	if test "$(uname)" = "Darwin"; then
```

Reply:
I'd like to point out that the script employs some features specific to Bash and not just POSIX shell, including the use of $(command -v ...). This Bash built-in is employed to check the availability of Python and pip:

```bash
if ! command -v python >/dev/null 2>&1 && ! command -v python3 >/dev/null 2>&1; then
```

Given that Bash is widely adopted and supports these features consistently, using a bash shebang helps ensure that the script behaves as expected across various environments. While some of these features might be present in other shells, the use of Bash-specific features in the script before my edits makes it reasonable to continue with the bash shebang.

If the author wishes to modify the rest of the scripts accordingly before this PR can be merged, they are welcome to do so. However, my suggestion is to maintain the bash shebang for the reasons outlined above.